### PR TITLE
BM-9 Comments Create & Destroy, UI Fixes/Changes

### PR DESCRIPTION
**Pull request by umair-khalid-brainx**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -76,5 +76,6 @@ end
 # Other gems added
 
 gem 'devise'
-gem "image_processing", ">= 1.2"
+gem "image_processing"
```
Comment (usama-javaid-brainx) [inline Gemfile:79]: why have you changed gem version?

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:user_id, :post_id, :text)
+  end
+
+  def set_variables
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
```
Comment (usama-javaid-brainx) [inline app/controllers/comments_controller.rb:24]: you should show flash messages for every action in all controllers

File: config/application.rb [hunk 3]
```diff
@@ -16,7 +16,8 @@ class Application < Rails::Application
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
     #
-    # config.time_zone = "Central Time (US & Canada)"
+    config.time_zone = 'Karachi'
```
Comment (usama-javaid-brainx) [inline config/application.rb:19]: why we need to set time zone here?

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -3,27 +3,57 @@
...
+        <h5 class="card-title d-inline fw-bold my-4">Comments Section</h5><br>
+        <div class="my-3">
+          <% post.comments.each do |comment| %>
+            <div class="comment-box">
+              <p class="card-text d-inline fw-bold my-3 fs-5"><%= comment.user.name %></p><br>
+              <p class="card-text d-inline my-3 fs-5 "><%= comment.text %></p><br>
+              <p class="card-text d-inline fs-6 fst-italic text-secondary my-3"><%= local_time_ago(comment.created_at) %></p><br>
+              <% if comment.user == current_user %>
+                <%= link_to "Delete Comment", user_post_comment_path(comment.user.id, comment.post.id, comment.id), data: { "turbo-method": :delete }, :class => "no-deco text-danger" %>
+              <% end %>
```
Comment (usama-javaid-brainx) [inline app/views/posts/index.html.erb:42]: you must try to use eager loading here

**Review by usama-javaid-brainx — CHANGES_REQUESTED** [review]

File: config/application.rb (hunk 3 above)
Comment (umair-khalid-brainx) [reply config/application.rb:19]: Time in my Rails app is 5 hours behind local, changing time solved nothing.

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (umair-khalid-brainx) [reply app/controllers/comments_controller.rb:24]: facing some issues showing notices with turbo

**Pull request merged**